clang/lib/APINotes/APINotesYAMLCompiler.cpp

```diff
@@ -421,9 +424,8 @@ struct Tag {
   std::optional<EnumConvenienceAliasKind> EnumConvenienceKind;
   std::optional<bool> SwiftCopyable;
   FunctionsSeq Methods;
+  TagsSeq Tags;
```

Comment:
This would be confusing as the reader might expect all child tags to be here.

Reply:
These are, in fact, child tags – I can add a comment here describing the meaning of this field.